### ISSUE TITLE
Core plugins inconsistent dependency on django CMS

### ISSUE THREAD

**Issue by yakky** [divio/meta#2]
Some core plugin has django CMS in its install_requires (e.g.: djangocms-pictues), while others don't (djangocms-link)
I think this should be more consistent
I propose that all plugins depends on django CMS (with appropriate versioning) and don't depend on Django

**Comment by mkoistinen** [comment]
I agree. All plugins (core or otherwise) should list django-cms as a dependency as they inherit from CMSPlugin and CMSPluginBase **by definition**. Depending on Django seems silly =).
